### PR TITLE
NEW unit tests for the Import & Export classes

### PR DESCRIPTION
**Pull request by carstingaxion**
### Description of the Change

~~Closes #~~

### How to test the Change

```bash
npm run test:unit:php -- --group migrate
```

### Changelog Entry

> Added - New unit tests for the Import & Export classes

### Credits

Props @carstingaxion @mauteri 

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/GatherPress/gatherpress/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Comment by carstingaxion** [comment]
Hello @mauteri ,

can you please help me here, please?

I'm wondering if the failing unit-test should be _fixed_ rather in `class-import.php` or maybe better in `class-event.php` as it could be helpful to check if the needed array keys exist before saving.

What do you think?

File: includes/core/classes/class-export.php [hunk 1]
```diff
@@ -96,6 +96,18 @@ public function prepare( WP_Post $post ): void {
...
+	 * Checks if the currently exported post is of type 'gatherpress_event'.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param  WP_Post $post Current meta key.
+	 * @return bool          True, when the currently exported post is of type 'gatherpress_event', false otherwise.
+	 */
+	protected function validate( WP_Post $post ): bool {
+		return ( Event::POST_TYPE === $post->post_type );
+	}
```
Comment (mauteri) [inline includes/core/classes/class-export.php:109]: Let's add this to the new `Validate` class. `Validate::event_post_type( $post )`

https://github.com/GatherPress/gatherpress/blob/develop/includes/core/classes/class-validate.php

File: test/unit/php/includes/core/classes/class-test-export.php [hunk 2]
```diff
@@ -40,6 +42,205 @@ public function test_setup_hooks(): void {
...
+			10,
+			has_action( 'the_post', array( $instance, 'prepare' ) ),
+			'Failed to assert that the "the_post" action was added.'
+		);
+
+		$this->assertSame(
+			10,
+			has_filter( 'wxr_export_skip_postmeta', array( $instance, 'extend' ) ),
+			'Failed to assert that the "wxr_export_skip_postmeta" filter was added.'
+		);
```
Comment (mauteri) [inline test/unit/php/includes/core/classes/class-test-export.php:76]: There's a helper to do this. Example [here](https://github.com/GatherPress/gatherpress/blob/develop/test/unit/php/includes/core/classes/class-test-block.php#L30).

**Review by mauteri — CHANGES_REQUESTED** [review]

File: includes/core/classes/class-export.php (hunk 1 above)
Comment (mauteri) [reply includes/core/classes/class-export.php]: never mind, we can use `Validate::event_post_id` here.

**Review by mauteri — APPROVED** [review]

**Pull request merged**